tools/schemacode/bidsschematools/render.py

```diff
@@ -1133,3 +1133,37 @@ def define_common_principles(schema, src_path=None):
     string = string.replace("SPEC_ROOT", get_relpath(src_path))
 
     return string
+
+
+def define_allowed_top_directories(schema, src_path=None) -> str:
```

Comment:
@tsalo not sure it makes much sense to add a test for that

Reply:
The problem is that RTD will still build if the rendering function is broken, so any kind of test would be helpful. Even just a smoke test to make sure that a string is returned (without checking the contents) would be useful.

Reply:
@tsalo added a very tiny smoke test